### ISSUE TITLE
driver-nuopc/cime_config/config_component.xm issue

### ISSUE THREAD

**Issue by lxu16** [E3SM-Project/E3SM#5250]
I tried to build the E3SM codes but met the following strange error.

ERROR: Makes no sense to have empty read-only file: /global/u1/l/lix011/E3SM_c20221012/driver-nuopc/cime_config/config_component.xml

I do not think the master has the driver-nupoc anymore.

Does anyone give some insights on this error?

**Comment by rljacob** [comment]
Yes E3SM doesn't have the nuopc driver.  You'll have to use the MCT one.

**Comment by lxu16** [comment]
How do I set the configuration to use the MCT one? I though the model in default uses the MCT one. No?

**Comment by rljacob** [comment]
Yes MCT is the default.  Do you have an environment variable called CIME_MODEL set to cesm?  That would change it.
